vars/Darwin.yml

```diff
@@ -0,0 +1,3 @@
+---
+
+manala_dnsmasq_configs_dir:       /usr/local/etc/dnsmasq.d
```

Comment:
Remove unused space charcters between "manala_dnsmasq_configs_dir:" and "/usr/local/etc/dnsmasq.d"